client2/src/game/gui/ClanPanel.js

```diff
@@ -0,0 +1,233 @@
+import PanelTemplate from "./PanelTemplate";
```

Comment:
The clan feature has been disabled for now. Some work for it was done a while ago but it was removed, due to being buggy and not finished.
This clan panel is not needed for now, as it will need to be remade when clans are added back.

Reply:
Removed at https://github.com/Arcanorum/dungeonz/pull/99/commits/559e46b0dab5d1a050e33f2b0faa246d76354a20